dss/subscriptions_v2/__init__.py

```diff
@@ -16,11 +16,25 @@ class SubscriptionData:
     FORM_FIELDS = 'form_fields'
     PAYLOAD_FORM_FIELD = 'payload_form_field'
     ATTACHMENTS = 'attachments'
+    STATS = 'stats'
+    BLANK_STATS = {"attempted": 0, "successful": 0, "failed": 0}
 
 
 subscription_db_table = f"dss-subscriptions-v2-{{}}-{os.environ['DSS_DEPLOYMENT_STAGE']}"
 
 
+def update_subscription_stats(doc: dict, status: bool):
```

Comment:
This method is not concurrency safe for DDB writes. Consider using a concurrency safe DDB update method such as [atomic counters](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/WorkingWithItems.html#WorkingWithItems.AtomicCounters), or [conditional writes](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/WorkingWithItems.html#WorkingWithItems.ConditionalUpdate).

Reply:
addressed